### ISSUE TITLE
No module named 'paddle.utils.cpp_extension'

### ISSUE THREAD

**Issue by No54** [PaddlePaddle/PaddleOCR#2892]
PaddleOCR, Dockerfile like: 

FROM registry.baidubce.com/paddlepaddle/paddle:2.0.0
...

When docker running, I met a bug No module named 'paddle.utils.cpp_extension'.

Please give some advise about this question, if any expert known. Thanks! ^_^

**Comment by No54** [comment]
> When do you meet this error? running the paddleocr program?

docker run ... after a few minutes，the container log show the error

**Comment by Huoran559** [comment]
/usr/local/lib/python3.7/site-packages/pkg_resources/_vendor/pyparsing.py:943: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  collections.MutableMapping.register(ParseResults)
/usr/local/lib/python3.7/site-packages/setuptools/depends.py:2: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
Traceback (most recent call last):
  File "/usr/local/bin/hub", line 5, in <module>
    from paddlehub.commands.utils import execute
  File "/usr/local/lib/python3.7/site-packages/paddlehub/__init__.py", line 31, in <module>
    from paddlehub import datasets
  File "/usr/local/lib/python3.7/site-packages/paddlehub/datasets/__init__.py", line 16, in <module>
    from paddlehub.datasets.chnsenticorp import ChnSentiCorp
  File "/usr/local/lib/python3.7/site-packages/paddlehub/datasets/chnsenticorp.py", line 19, in <module>
    from paddlehub.datasets.base_nlp_dataset import TextClassificationDataset
  File "/usr/local/lib/python3.7/site-packages/paddlehub/datasets/base_nlp_dataset.py", line 21, in <module>
    import paddlenlp
  File "/usr/local/lib/python3.7/site-packages/paddlenlp/__init__.py", line 20, in <module>
    from . import ops
  File "/usr/local/lib/python3.7/site-packages/paddlenlp/ops/__init__.py", line 15, in <module>
    from .transformer.decoding import *
  File "/usr/local/lib/python3.7/site-packages/paddlenlp/ops/transformer/decoding.py", line 24, in <module>
    from paddlenlp.ops.ext_utils import load
  File "/usr/local/lib/python3.7/site-packages/paddlenlp/ops/ext_utils.py", line 25, in <module>
    from paddle.utils.cpp_extension import load_op_meta_info_and_register_op
ModuleNotFoundError: No module named 'paddle.utils.cpp_extension'

**Comment by Huoran559** [comment]
command line was used:
`docker run -dp 8868:8868 --name paddle_ocr paddleocr:cpu`

output:
`/usr/local/lib/python3.7/site-packages/pkg_resources/_vendor/pyparsing.py:943: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  collections.MutableMapping.register(ParseResults)
/usr/local/lib/python3.7/site-packages/setuptools/depends.py:2: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
Traceback (most recent call last):
  File "/usr/local/bin/hub", line 5, in <module>
    from paddlehub.commands.utils import execute
  File "/usr/local/lib/python3.7/site-packages/paddlehub/__init__.py", line 31, in <module>
    from paddlehub import datasets
  File "/usr/local/lib/python3.7/site-packages/paddlehub/datasets/__init__.py", line 16, in <module>
    from paddlehub.datasets.chnsenticorp import ChnSentiCorp
  File "/usr/local/lib/python3.7/site-packages/paddlehub/datasets/chnsenticorp.py", line 19, in <module>
    from paddlehub.datasets.base_nlp_dataset import TextClassificationDataset
  File "/usr/local/lib/python3.7/site-packages/paddlehub/datasets/base_nlp_dataset.py", line 21, in <module>
    import paddlenlp
  File "/usr/local/lib/python3.7/site-packages/paddlenlp/__init__.py", line 20, in <module>
    from . import ops
  File "/usr/local/lib/python3.7/site-packages/paddlenlp/ops/__init__.py", line 15, in <module>
    from .transformer.decoding import *
  File "/usr/local/lib/python3.7/site-packages/paddlenlp/ops/transformer/decoding.py", line 24, in <module>
    from paddlenlp.ops.ext_utils import load
  File "/usr/local/lib/python3.7/site-packages/paddlenlp/ops/ext_utils.py", line 25, in <module>
    from paddle.utils.cpp_extension import load_op_meta_info_and_register_op
ModuleNotFoundError: No module named 'paddle.utils.cpp_extension'
`

**Comment by phattruongai** [comment]
I also met this problem, but in the build Dockerfile step:

```
FROM nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04

RUN apt-get update && \
    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
    fakeroot \
    ca-certificates \
    dpkg-dev \
    g++ \
    python3-dev \
    curl \
    openjdk-8-jdk-headless
    
RUN apt-get update && apt-get install -y python3-pip && \
    pip3 install --upgrade pip && pip3 install awscli && \
    apt install -y libsm6 libxext6 && \
    apt-get install -y libxrender-dev && \
    pip3 install paddlepaddle-gpu==2.0.0 

RUN mkdir /working_directory && cd /working_directory && ls -l

RUN pip3 install --upgrade paddlehub==2.0.0

RUN apt-get install -y git && git clone https://github.com/PaddlePaddle/PaddleOCR.git && \
    cd PaddleOCR && mkdir inference_model && ls -l && pip3 install -r requirements.txt && \
    rm -r deploy/hubserving/ocr_system

WORKDIR /PaddleOCR

RUN ls -l
ADD ./ocr_system  deploy/hubserving/ocr_system

ENV CUDA_VISIBLE_DEVICES=0
RUN hub install deploy/hubserving/ocr_system/
...
```

It raised this error at the command `hub install deploy/hubserving/ocr_system/`

```
Step 16/25 : RUN hub install deploy/hubserving/ocr_system/
 ---> Running in 2afba6f23348
/usr/local/lib/python3.6/dist-packages/paddle/fluid/framework.py:298: UserWarning: You are using GPU version Paddle, but your CUDA device is not set properly. CPU device will be used by default.
  "You are using GPU version Paddle, but your CUDA device is not set properly. CPU device will be used by default."
/usr/lib/python3/dist-packages/setuptools/depends.py:2: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
Traceback (most recent call last):
  File "/usr/local/bin/hub", line 5, in <module>
    from paddlehub.commands.utils import execute
  File "/usr/local/lib/python3.6/dist-packages/paddlehub/__init__.py", line 24, in <module>
    from paddlehub import datasets
  File "/usr/local/lib/python3.6/dist-packages/paddlehub/datasets/__init__.py", line 18, in <module>
    from paddlehub.datasets.chnsenticorp import ChnSentiCorp
  File "/usr/local/lib/python3.6/dist-packages/paddlehub/datasets/chnsenticorp.py", line 19, in <module>
    from paddlehub.datasets.base_nlp_dataset import TextClassificationDataset
  File "/usr/local/lib/python3.6/dist-packages/paddlehub/datasets/base_nlp_dataset.py", line 23, in <module>
    from paddlenlp.transformers import PretrainedTokenizer
  File "/usr/local/lib/python3.6/dist-packages/paddlenlp/__init__.py", line 20, in <module>
    from . import ops
  File "/usr/local/lib/python3.6/dist-packages/paddlenlp/ops/__init__.py", line 15, in <module>
    from .transformer.decoding import *
  File "/usr/local/lib/python3.6/dist-packages/paddlenlp/ops/transformer/decoding.py", line 24, in <module>
    from paddlenlp.ops.ext_utils import load
  File "/usr/local/lib/python3.6/dist-packages/paddlenlp/ops/ext_utils.py", line 25, in <module>
    from paddle.utils.cpp_extension import load_op_meta_info_and_register_op
ModuleNotFoundError: No module named 'paddle.utils.cpp_extension'
The command '/bin/sh -c hub install deploy/hubserving/ocr_system/' returned a non-zero code: 1
```

**Comment by Huoran559** [comment]
I have work around to fix it, just install paddlepaddle==2.1.0 in Dockerfile

`
# Version: 2.0.0
FROM registry.baidubce.com/paddlepaddle/paddle:2.0.0


# PaddleOCR base on Python3.7
RUN pip3.7 install --upgrade pip -i https://mirror.baidu.com/pypi/simple

RUN pip3.7 install paddlepaddle==2.1.0 -i https://mirror.baidu.com/pypi/simple

RUN pip3.7 install paddlehub --upgrade -i https://mirror.baidu.com/pypi/simple

RUN git clone https://github.com/PaddlePaddle/PaddleOCR.git /PaddleOCR

`

**Comment by phattruongai** [comment]
Yes, I've just fixed it too , seem like version 2.0.0 have some problems
